src/core/backend/core.rs

```diff
@@ -82,6 +85,15 @@ pub(crate) trait BackendStorage: Debug {
     async fn exist(&self, op: Operation, keys: &[BobKey]) -> Result<Vec<bool>, Error>;
     async fn exist_alien(&self, op: Operation, keys: &[BobKey]) -> Result<Vec<bool>, Error>;
 
+    async fn run(&self) -> Result<()> {
+        BACKEND_STATE.value(BACKEND_STARTING);
+        let result = self.run_backend().await;
+        BACKEND_STATE.value(BACKEND_STARTED);
+        result
+    }
+
+    fn spawn_counter(&self) {}
```

Comment:
Why do you think this func belongs in BackendStorage trait?

Reply:
First of all I thought that it would be good to have background workers in one place. To spawn blobs (or not blobs) counter I need to do next steps:
1. Call counter from `grinder`  (from method `run_periodic_tasks`) using `backend` method
2. Then backend somehow need to spawn counter, BUT it has storage field of `dyn BackendStorage` type, so it can only use trait methods to spawn counter of storage.

So I found only 2 solutions:
1. Make trait method with empty default implementation, to call it from `Backend` (and from `Grinder`; so every storage may spawn it's counters)
2. Make this method on `Pearl` level, but in this case it seems like we should explicitly spawn counter from binary (that's not a problem, but makes us to change the code in binary either)